cmd/retry-storage.go

```diff
@@ -264,11 +277,10 @@ func (f *retryStorage) reInitUponDiskNotFound(err error) bool {
 // attempts three times before giving up.
```

Comment:
IIUC, `reInit` method attempts to connect and load format only 2 times. `f.maxRetryAttempts` is initialized to 1 and `newRetrySimple` starts from 0.